architecture-diagrams/C1.puml

```diff
@@ -0,0 +1,33 @@
+@startuml Book Club Hub App
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
```

Comment:
That's a great suggestion but unfortunately I've never created any bash scripts. I've have a look online, maybe someone created a package like this for python and I could use it :)

Reply:
you could use ready docker image :) 
https://hub.docker.com/r/dstockhammer/plantuml

docker run --rm -v $(pwd):/data dstockhammer/plantuml architecture-diagrams/C1.puml
docker run --rm -v $(pwd):/data dstockhammer/plantuml architecture-diagrams/C3-api.puml

for now it is enough to put it in README :)

Reply:
later you could create github action which will verify if something was changed in *.puml files. If something was changed then you wont be able to make git push until MR is not with updated images.
It is not something very needed but it could be fun :)

Reply:
In case of any problems with running docker commands we can have call :)

Reply:
I see it works also for directory 
```
docker run --rm -v $(pwd):/data dstockhammer/plantuml architecture-diagrams/  
```